README.md

```diff
@@ -558,6 +558,8 @@ const uiSchema = {
 };
 ```
 
+In addition it is valid to mark an array as "required".  This means the user must select at least one item to add to the array.
```

Comment:
Yep.  More than happy to leave the array case out.  If we can achieve the same result using minItem and it's more standards compliant I can't see why we wouldn't.